socless/jinja.py

```diff
@@ -90,8 +92,34 @@ def env(env_var_name: str) -> str:
         raise SoclessBootstrapError(f"Environment Variable {env_var_name} not found")
 
 
+def secret(secret_path: str) -> str:
+    """Custom Jinja function/filter that fetches a secret from SSM Parameter Store.
+    Args:
+        secret_path: "/socless/slack/bot_token"
+    """
+    return fetch_from_ssm(secret_path)
+
+
+def fromtimestamp(timestamp: Union[int, str], tz: str = "UTC") -> str:
+    """Custom Jinja function/filter that converst an epoch timestamp to an ISO datetime (yyyy-mm-ddThh:mm:ss). Timezone sticks with the system's timezone.
+    Args:
+        timestamp (str | int): an epoch timestamp
+        tz: (str): an optional argument that's default to UTC. If provided, tz value must follow be one of the timezone names exists in "pytz.all_timezones". Here're some useful resources:
+            - https://en.wikipedia.org/wiki/List_of_tz_database_time_zones
+            - https://gist.github.com/heyalexej/8bf688fd67d7199be4a1682b3eec7568
+    """
+    tzinfo = timezone(tz)
+    return datetime.fromtimestamp(int(timestamp), tz=tzinfo).isoformat()
```

Comment:
If a user passes a bad string to fromtimestamp, the stacktrace doesn't easily show where/how this error occurred:
```
ValueError: invalid literal for int() with base 10: '
```

Can we wrap the int() call in a try/except ValueError and surface a descriptive `SoclessBootstrapError` to help users debug this faster?